### DISCUSSION TITLE
A question about the 'cell_labels' in data files.

### DISCUSSION THREAD

**Discussion by CocoGzh** [brianhie/scanorama#99] [Q&A] [answered]
I have downloaded the whole data from http://cb.csail.mit.edu/cb/scanorama/data.tar.gz. I'm a little confused about how to use it. 
It seems that the 'cell_labels' file is a annotation for other files. For example, for 'pancreas' file, it contains 5 datasets. In what order are these 5 datasets combined to correspond to the labels in ‘pancreas_cluster’ file?

**Comment by brianhie** [comment, marked as answer]
Hi @CocoGzh, those data files are for reproducing the results from the original paper. E.g., for pancreas, the labels are used here: https://github.com/brianhie/scanorama/blob/master/bin/pancreas.py#L39
